scripts/migration/menas_db.py

```diff
@@ -98,37 +98,37 @@ def ensure_collections_exist(collection_names: List[str]) -> None:
         # deliberately only checking migrating collections, not all (landing_page_statistics may not be present)
         return ensure_collections_exist(DATA_MIGRATING_COLLECTIONS)
 
-    def get_distinct_ds_names_from_ds_names(self, ds_names: List[str], not_locked_only: bool) -> List[str]:
-        return self.get_distinct_entities_ids(ds_names, DATASET_COLLECTION, not_locked_only)
+    def get_distinct_ds_names_from_ds_names(self, ds_names: List[str], migration_free_only: bool) -> List[str]:
```

Comment:
`migration_free_only` sounds, for some reason, very funny to me. `not_migrated_only` or `non_migrated_only` sounds more standard. Not pushing it.